index.html

```diff
@@ -2891,13 +2891,11 @@ <h3>Production</h3>
             <p>
 The <a href="https://www.w3.org/TR/json-ld11/#the-context">JSON-LD
 specification</a> defines values that are valid for this entry. This entry
-contains representation-specific syntax and therefore could be present in the <a
-href="#data-model">data model</a> to aid in lossless conversion. If the entry
-is present in the <a href="#data-model">data model</a>, it MUST be used during
-production unless either an alternative <code>@context</code> value is
-explicitly provided to the producer or if the value from the data model is not
-valid according to the consumption rules.
```

Comment:
Why remove the requirement to check for validity of the property? Without this, a conforming resolver would be susceptible to injection of bad context values that it would blindly pass through. This seems like a gaping security and interoperability hole.

Reply:
@jricher you want `or if the value from the data model is not valid according to the consumption rules.`
to be added back? that would require it to be in the data model... perhaps you would prefer that the context provided to the json-ld production must be valid according to the consumption rules?

I